chia/consensus/blockchain.py

```diff
@@ -270,11 +270,18 @@ async def run_single_block(self, block: FullBlock, fork_info: ForkInfo) -> None:
         additions: list[tuple[Coin, Optional[bytes]]] = []
         removals: list[tuple[bytes32, Coin]] = []
         if block.transactions_generator is not None:
+            prev_tx_height = prev_tx_block(
```

Comment:
nit but i would move this right before the call to get_flags_for_height_and_constants where it is used